src/Backend/SgateShopgatePlugin/Components/User.php

```diff
@@ -165,37 +165,52 @@ public function loginUser($request, $httpResponse)
     public function getUser($request)
     {
         try {
-            $decoded    = $this->webCheckoutHelper->getJWT($request->getCookie('token'));
-            $customerId = $decoded['customer_id'];
+            $decoded = $this->webCheckoutHelper->getJWT($request->getCookie('token'));
 
-            $sql = ' SELECT password, email FROM s_user WHERE customernumber = ? AND active=1 AND (lockeduntil < now() OR lockeduntil IS NULL) ';
-            $user = Shopware()->Db()->fetchRow($sql, array($customerId)) ? : array();
+            if ($decoded['error']) {
```

Comment:
isset(), or if you don't want to fix all the $decoded related stuff, just add 'error' => false as default to the getJWT method and do $decoded['error'] === true